### DISCUSSION TITLE
Is there a way to recover comments made to a file if a repository is updated, or are those comments lost forever?

### DISCUSSION THREAD

**Discussion by campbelle1** [ReviewNB/support#121] [Q&A] [answered]
I had a bunch of comments I made on a colleague's python notebooks. The repository that housed these notebooks was updated (I don't think the notebooks themselves were altered though). I went onto ReviewNB and the comments are no longer there. Is there a way to get them back?

**Comment by amit1rrr** [comment, marked as answer]
So [JDoc comments](https://blog.reviewnb.com/commenting-for-jupyter/) are tied to owner login, repository name & notebook path (including filename), if any of these changes then the file is treated as new & the old comments are not displayed on it.

But your comments are still available in the database and will render immediately as soon as they've the same combination of (owner login, repository name & notebook path). Is it possible for you to revert to the old repository state / file path? If not, let us know the details (email to support@reviewnb.com) & we can migrate the comments for you.
